netket/utils.py

```diff
@@ -47,6 +47,11 @@ def jit_if_singleproc(f, *args, **kwargs):
 else:
     mpi4jax_available = False
 
+if mpi4jax_available and not mpi4jax.__version__ >= "0.2.7":
```

Comment:
You sure about that? 
https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python